src/Payments.sol

```diff
@@ -608,15 +598,13 @@ contract Payments is
             }
 
             require(
-                newRate <= oldRate,
-                "failed to modify rail: cannot increase rate on terminated rail"
+                newRate == oldRate,
+                "failed to modify rail: cannot change rate on terminated rail"
             );
         } else {
-            validateRateChangeRequirementsForNonTerminatedRail(
-                rail,
-                payer,
-                oldRate,
-                newRate
+            require(
+                isAccountLockupFullySettled(payer) || newRate == oldRate,
```

Comment:
same as above.

Reply:
This I'm fine with (for now). I _want_ to be able to reduce the lockup in this case, but I think we can live with this.